### PR TITLE
cleanup redirects

### PR DESCRIPTION
**Pull request by eshaben**
This PR removes all the old redirects. I went through each redirect to make sure that the link was not getting any traffic and that there were no associated backlinks, or if there were backlinks, there weren't a high number of backlinks or the backlinks were of quality (i.e., no spam sites or no random urls like 22adsh82.com). 

I documented the remaining redirects and their associated backlinks in [this spreadsheet](https://docs.google.com/spreadsheets/d/1-BoQmAjH9v8r1ohhujUSfU9n7QdgORfJNS5agJFRq64/edit?usp=sharing) so we can decide if we want to update some of the medium articles we've published or just leave the redirects so all the links work.

### CODE REVIEW

File: mkdocs-cn/mkdocs.yml [hunk 1]
```diff
@@ -53,203 +53,16 @@
...
-        'builders/build/canonical-contracts/precompiles/staking.md': 'builders/pallets-precompiles/precompiles/staking.md'
-        'builders/build/canonical-contracts/precompiles/democracy.md': 'builders/pallets-precompiles/precompiles/index.md'
-        'builders/build/canonical-contracts/precompiles/erc20.md': 'builders/pallets-precompiles/precompiles/erc20.md'
-        'builders/build/canonical-contracts/precompiles/eth-mainnet.md': 'builders/pallets-precompiles/precompiles/eth-mainnet.md'
-        'builders/build/canonical-contracts/contracts.md': 'builders/build/canonical-contracts.md'
-        'builders/pallets-precompiles/precompiles/democracy.md': 'builders/pallets-precompiles/precompiles/index.md'
-        'builders/integrations/bridges/axelar.md': 'builders/interoperability/protocols/axelar.md'
-        'builders/integrations/bridges/index.md': 'builders/interoperability/protocols/index.md'
-        'builders/xcm/fees.md': 'builders/interoperability/xcm/fees.md'
+        'builders/get-started/networks/layer2/bobabeam.md': 'builders/get-started/networks/index.md'
```
Comment (themacexpert) [inline mkdocs-cn/mkdocs.yml:57]:
```suggestion
        'builders/get-started/networks/layer2/bobabeam.md': 'builders/get-started/networks/'
```

Comment (themacexpert) [reply mkdocs-cn/mkdocs.yml:57]: Not sure if this is the right formatting but https://docs.moonbeam.network/builders/get-started/networks/index/ 404s

File: mkdocs-cn/mkdocs.yml [hunk 2]
```diff
@@ -53,203 +53,16 @@
...
+        'builders/interoperability/xcm/xc-integration.md': 'builders/interoperability/xcm/xc-registration/index.md'
```
Comment (themacexpert) [inline mkdocs-cn/mkdocs.yml:58]: Same story as above

File: mkdocs.yml [hunk 3]
```diff
@@ -53,476 +53,36 @@
...
-              'builders/build/canonical-contracts/precompiles/erc20.md': 'builders/pallets-precompiles/precompiles/erc20.md'
-              'builders/build/canonical-contracts/precompiles/eth-mainnet.md': 'builders/pallets-precompiles/precompiles/eth-mainnet.md'
-              'builders/build/canonical-contracts/contracts.md': 'builders/build/canonical-contracts.md'
-              'builders/pallets-precompiles/precompiles/democracy.md': 'builders/pallets-precompiles/precompiles/index.md'
-              'builders/integrations/bridges/axelar.md': 'builders/interoperability/protocols/axelar.md'
-              'builders/integrations/bridges/hyperlane.md': 'builders/interoperability/protocols/hyperlane.md'
-              'builders/integrations/bridges/index.md': 'builders/interoperability/protocols/index.md'
-              'builders/integrations/bridges/layerzero.md': 'builders/interoperability/protocols/layerzero.md'
-              'builders/integrations/bridges/wormhole.md': 'builders/interoperability/protocols/wormhole.md'
+              'builders/get-started/networks/layer2/bobabeam.md': 'builders/get-started/networks/index.md'
```
Comment (themacexpert) [inline mkdocs.yml:60]: 404s

File: mkdocs.yml [hunk 4]
```diff
@@ -53,476 +53,36 @@
...
+              'builders/interact/hardhat.md': 'builders/build/eth-api/dev-env/hardhat.md'
+              'builders/interoperability/xcm/xc-integration.md': 'builders/interoperability/xcm/xc-registration/index.md'
```
Comment (themacexpert) [inline mkdocs.yml:62]: 404s

File: mkdocs.yml [hunk 5]
```diff
@@ -53,476 +53,36 @@
...
-              'ru/staking/overview.md': 'learn/features/staking.md'
-              'ru/staking/stake.md': 'tokens/staking/stake.md'
-              'ru/staking/stakingprecompile.md': 'builders/pallets-precompiles/precompiles/staking.md'
-              'ru/governance/voting.md': 'tokens/governance/voting.md'
-              'ru/governance/overview.md': 'learn/features/governance.md'
-              'ru/governance/proposals.md': 'tokens/governance/proposals.md'
-              'ru/treasury/overview.md': 'learn/features/treasury.md'
-              'ru/overview/why-polkadot.md': 'learn/platform/why-polkadot.md'
-              'ru/overview/vision-features.md': 'learn/platform/vision.md'
+              'builders/xcm/xc-integration.md': 'builders/interoperability/xcm/xc-registration/index.md'
```
Comment (themacexpert) [inline mkdocs.yml:69]: same link as earlier that 404s

File: mkdocs.yml [hunk 6]
```diff
@@ -53,476 +53,36 @@
...
+              'getting-started/using-metamask.md': 'tokens/connect/metamask.md'
+              'getting-started/using-remix.md': 'builders/build/eth-api/dev-env/remix.md'
+              'getting-started/using-truffle.md': 'builders/build/eth-api/dev-env/truffle.md'
+              'getting-started/local-node/deploy-contract.md': 'builders/build/eth-api/libraries/index.md'
+              'getting-started/local-node/using-remix.md': 'builders/build/eth-api/dev-env/remix.md'
+              'getting-started/local-node/web3-transaction.md': 'builders/build/eth-api/libraries/web3py.md'
+              'getting-started/moonriver/integrate-metamask.md': 'builders/get-started/networks/moonriver.md'
+              'getting-started/testnet/faucet.md': 'builders/get-started/networks/moonbase.md'
+              'getting-started/testnet/metamask.md': 'tokens/connect/metamask.md'
+              'getting-started/testnet/precompiles.md': 'builders/pallets-precompiles/precompiles/index.md'
```
Comment (themacexpert) [inline mkdocs.yml:79]: 404s

File: mkdocs.yml [hunk 7]
```diff
@@ -53,476 +53,36 @@
...
-              'ru/governance/overview.md': 'learn/features/governance.md'
-              'ru/governance/proposals.md': 'tokens/governance/proposals.md'
-              'ru/treasury/overview.md': 'learn/features/treasury.md'
-              'ru/overview/why-polkadot.md': 'learn/platform/why-polkadot.md'
-              'ru/overview/vision-features.md': 'learn/platform/vision.md'
+              'builders/xcm/xc-integration.md': 'builders/interoperability/xcm/xc-registration/index.md'
+              'getting-started/using-metamask.md': 'tokens/connect/metamask.md'
+              'getting-started/using-remix.md': 'builders/build/eth-api/dev-env/remix.md'
+              'getting-started/using-truffle.md': 'builders/build/eth-api/dev-env/truffle.md'
+              'getting-started/local-node/deploy-contract.md': 'builders/build/eth-api/libraries/index.md'
```
Comment (themacexpert) [inline mkdocs.yml:73]: 404s

**Review by themacexpert — CHANGES_REQUESTED** [review]
Seems like the new site doesn't like the index.md pages

File: mkdocs.yml (hunk 7 above)
Comment (themacexpert) [reply mkdocs.yml:73]: builders/build/eth-api/libraries/index.md is what I'm referring to

File: mkdocs.yml (hunk 6 above)
Comment (themacexpert) [reply mkdocs.yml:79]: builders/pallets-precompiles/precompiles/index.md

File: mkdocs.yml (hunk 4 above)
Comment (themacexpert) [reply mkdocs.yml:62]: builders/interoperability/xcm/xc-registration/index.md

**Comment by eshaben** [comment]
@themacexpert I'm not seeing what you're seeing. When I go to `http://127.0.0.1:8000/builders/get-started/networks/layer2/bobabeam` it automatically redirects to `http://127.0.0.1:8000/builders/get-started/networks/`

**Comment by themacexpert** [comment]
Ignore my comments, all is well!

**Review by themacexpert — APPROVED** [review]
LGTM 👍

**Pull request merged**